CHANGELOG

```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Updated
 
+- Bump DS version to 3.4.0
 - Bump nvmrc version to Node 20 and DS package to release candidate 3.4.0-rc.
```

Comment:
I think you it's ok to remove "and DS package to release candidate 3.4.0-rc." since this should not really go into `production` and was used for testing. So you can leave the 3.4.0 message above.